### ISSUE TITLE
Load Issue with Julia v1.3-alpha

### ISSUE THREAD

**Issue by cj-mclaughlin** [JuliaInterop/Cxx.jl#435]
Hello, I am having issues importing Cxx with Julia version 1.3 alpha. When I run `using Cxx`, here is the error that occurs:

```
julia> using Cxx
[ Info: Precompiling Cxx [a0b5b9ef-44b7-5148-a2d1-f6db19f3c3d2]
free(): invalid pointer

signal (6): Aborted
in expression starting at /home/connor/.julia/packages/Cxx/vxYtJ/src/Cxx.jl:178
__libc_signal_restore_set at /build/glibc-OTsEL5/glibc-2.27/signal/../sysdeps/unix/sysv/linux/nptl-signals.h:80 [inlined]
raise at /build/glibc-OTsEL5/glibc-2.27/signal/../sysdeps/unix/sysv/linux/raise.c:48
abort at /build/glibc-OTsEL5/glibc-2.27/stdlib/abort.c:79
__libc_message at /build/glibc-OTsEL5/glibc-2.27/libio/../sysdeps/posix/libc_fatal.c:181
malloc_printerr at /build/glibc-OTsEL5/glibc-2.27/malloc/malloc.c:5350
_int_free at /build/glibc-OTsEL5/glibc-2.27/malloc/malloc.c:4157 [inlined]
__libc_free at /build/glibc-OTsEL5/glibc-2.27/malloc/malloc.c:3124
_ZN4llvm19SmallPtrSetImplBase4GrowEj at /home/connor/Documents/julia-1.3.0-alpha/bin/../lib/julia/libLLVM-6.0.so (unknown line)
_ZN4llvm19SmallPtrSetImplBase14insert_imp_bigEPKv at /home/connor/Documents/julia-1.3.0-alpha/bin/../lib/julia/libLLVM-6.0.so (unknown line)
_ZN4llvm11LLVMContext9addModuleEPNS_6ModuleE at /home/connor/Documents/julia-1.3.0-alpha/bin/../lib/julia/libLLVM-6.0.so (unknown line)
_ZN4llvm6ModuleC1ENS_9StringRefERNS_11LLVMContextE at /home/connor/Documents/julia-1.3.0-alpha/bin/../lib/julia/libLLVM-6.0.so (unknown line)
finish_clang_init at /home/connor/.julia/packages/Cxx/vxYtJ/src/../deps/usr/lib/libcxxffi.so (unknown line)
init_clang_instance at /home/connor/.julia/packages/Cxx/vxYtJ/src/../deps/usr/lib/libcxxffi.so (unknown line)
#setup_instance#20 at /home/connor/.julia/packages/Cxx/vxYtJ/src/initialization.jl:39
#setup_instance at ./tuple.jl:0 [inlined]
__init__ at /home/connor/.julia/packages/Cxx/vxYtJ/src/initialization.jl:427
_jl_invoke at /buildworker/worker/package_linux64/build/src/gf.c:2049 [inlined]
jl_apply_generic at /buildworker/worker/package_linux64/build/src/gf.c:2213
jl_apply at /buildworker/worker/package_linux64/build/src/julia.h:1630 [inlined]
do_call at /buildworker/worker/package_linux64/build/src/interpreter.c:328
eval_value at /buildworker/worker/package_linux64/build/src/interpreter.c:417
eval_stmt_value at /buildworker/worker/package_linux64/build/src/interpreter.c:368 [inlined]
eval_body at /buildworker/worker/package_linux64/build/src/interpreter.c:778
jl_interpret_toplevel_thunk_callback at /buildworker/worker/package_linux64/build/src/interpreter.c:888
unknown function (ip: 0xfffffffffffffffe)
unknown function (ip: 0x7f2f4a06698f)
unknown function (ip: 0xffffffffffffffff)
jl_interpret_toplevel_thunk at /buildworker/worker/package_linux64/build/src/interpreter.c:897
jl_toplevel_eval_flex at /buildworker/worker/package_linux64/build/src/toplevel.c:814
jl_eval_module_expr at /buildworker/worker/package_linux64/build/src/toplevel.c:181
jl_toplevel_eval_flex at /buildworker/worker/package_linux64/build/src/toplevel.c:640
jl_eval_module_expr at /buildworker/worker/package_linux64/build/src/toplevel.c:181
jl_toplevel_eval_flex at /buildworker/worker/package_linux64/build/src/toplevel.c:640
jl_parse_eval_all at /buildworker/worker/package_linux64/build/src/ast.c:873
jl_load at /buildworker/worker/package_linux64/build/src/toplevel.c:878
include at ./boot.jl:328 [inlined]
include_relative at ./loading.jl:1094
include at ./Base.jl:31
_jl_invoke at /buildworker/worker/package_linux64/build/src/gf.c:2043 [inlined]
jl_apply_generic at /buildworker/worker/package_linux64/build/src/gf.c:2213
jl_apply at /buildworker/worker/package_linux64/build/src/julia.h:1630 [inlined]
do_call at /buildworker/worker/package_linux64/build/src/interpreter.c:328
eval_value at /buildworker/worker/package_linux64/build/src/interpreter.c:417
eval_stmt_value at /buildworker/worker/package_linux64/build/src/interpreter.c:368 [inlined]
eval_body at /buildworker/worker/package_linux64/build/src/interpreter.c:778
eval_body at /buildworker/worker/package_linux64/build/src/interpreter.c:705
jl_interpret_toplevel_thunk_callback at /buildworker/worker/package_linux64/build/src/interpreter.c:888
unknown function (ip: 0xfffffffffffffffe)
unknown function (ip: 0x7f2f4965a68f)
unknown function (ip: 0x2)
jl_interpret_toplevel_thunk at /buildworker/worker/package_linux64/build/src/interpreter.c:897
jl_toplevel_eval_flex at /buildworker/worker/package_linux64/build/src/toplevel.c:814
jl_toplevel_eval_in at /buildworker/worker/package_linux64/build/src/toplevel.c:843
eval at ./boot.jl:330 [inlined]
eval at ./client.jl:433
_jl_invoke at /buildworker/worker/package_linux64/build/src/gf.c:2043 [inlined]
jl_apply_generic at /buildworker/worker/package_linux64/build/src/gf.c:2213
top-level scope at ./none:3
jl_toplevel_eval_flex at /buildworker/worker/package_linux64/build/src/toplevel.c:808
jl_toplevel_eval_flex at /buildworker/worker/package_linux64/build/src/toplevel.c:764
jl_toplevel_eval_in at /buildworker/worker/package_linux64/build/src/toplevel.c:843
eval at ./boot.jl:330
_jl_invoke at /buildworker/worker/package_linux64/build/src/gf.c:2043 [inlined]
jl_apply_generic at /buildworker/worker/package_linux64/build/src/gf.c:2213
exec_options at ./client.jl:271
_start at ./client.jl:468
_jl_invoke at /buildworker/worker/package_linux64/build/src/gf.c:2043 [inlined]
jl_apply_generic at /buildworker/worker/package_linux64/build/src/gf.c:2213
jl_apply at /buildworker/worker/package_linux64/build/ui/../src/julia.h:1630 [inlined]
true_main at /buildworker/worker/package_linux64/build/ui/repl.c:96
main at /buildworker/worker/package_linux64/build/ui/repl.c:217
__libc_start_main at /build/glibc-OTsEL5/glibc-2.27/csu/../csu/libc-start.c:310
_start at /home/connor/Documents/julia-1.3.0-alpha/bin/julia (unknown line)
Allocations: 1666512 (Pool: 1666085; Big: 427); GC: 2
ERROR: Failed to precompile Cxx [a0b5b9ef-44b7-5148-a2d1-f6db19f3c3d2] to /home/connor/.julia/compiled/v1.3/Cxx/ESGkI.ji.
Stacktrace:
 [1] error(::String) at ./error.jl:33
 [2] compilecache(::Base.PkgId, ::String) at ./loading.jl:1253
 [3] _require(::Base.PkgId) at ./loading.jl:1013
 [4] require(::Base.PkgId) at ./loading.jl:911
 [5] require(::Module, ::Symbol) at ./loading.jl:906
```

**Comment by Gnimuc** [comment]
At least, you need this patch https://github.com/JuliaInterop/Cxx.jl/pull/407

**Comment by ianshmean** [comment]
What's currently preventing the patch being merged and released? It'd be great to get Cxx stable on >=1.3

**Comment by Gnimuc** [comment]
lack of code reviewers, I think.
